.github/workflows/check.yml

```
@@ -13,14 +13,30 @@ permissions:
-  run:
-    working-directory: client
-
 jobs:
+  analyze-server:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    defaults:
+      run:
+        working-directory: server
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)